gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -10,18 +11,43 @@
 from gooddata_sdk.compute.model.metric import Metric
 
 
+@dataclass
+class TotalDimension:
+    idx: int
+    """index of dimension in which to calculate the total"""
+
+    items: list[str] = field(default_factory=list)
+    """items to use during total calculation"""
+
+
+@dataclass
```

Comment:
Please use attrs instead of built-in data classes.

Reply:
fixed in last commit